NorthwindCRUD/QueryBuilder/QueryExecutor.cs

```diff
@@ -0,0 +1,263 @@
+                "equals"                => Expression.Equal(field, searchValue),
+                "doesNotEqual"          => Expression.NotEqual(field, searchValue),
+                "inQuery"               => BuildInExpression(db, filter.SearchTree, field),
+                "notInQuery"            => Expression.Not(BuildInExpression(db, filter.SearchTree, field)),
+                "contains"              => CallContains(field, searchValue),
+                "doesNotContain"        => Expression.Not(CallContains(field, searchValue)),
+                "startsWith"            => CallStartsWith(field, searchValue),
+                "endsWith"              => CallEndsWith(field, searchValue),
+                "greaterThan"           => Expression.GreaterThan(field, searchValue),
+                "lessThan"              => Expression.LessThan(field, searchValue),
+                "greaterThanOrEqualTo"  => Expression.GreaterThanOrEqual(field, searchValue),
+                "lessThanOrEqualTo"     => Expression.LessThanOrEqual(field, searchValue),
+                "before"                => Expression.LessThan(CallCompare(field, searchValue), Expression.Constant(0)),
+                "after"                 => Expression.GreaterThan(CallCompare(field, searchValue), Expression.Constant(0)),
+                "today"                 => CallStartsWith(field, now.ToString("yyyy-MM-dd", CultureInfo.InvariantCulture)),
+                "yesterday"             => CallStartsWith(field, now.AddDays(-1).ToString("yyyy-MM-dd", CultureInfo.InvariantCulture)),
+                "thisMonth"             => CallStartsWith(field, now.ToString("yyyy-MM", CultureInfo.InvariantCulture)),
+                "lastMonth"             => CallStartsWith(field, now.AddMonths(-1).ToString("yyyy-MM", CultureInfo.InvariantCulture)),
+                "nextMonth"             => CallStartsWith(field, now.AddMonths(1).ToString("yyyy-MM", CultureInfo.InvariantCulture)),
+                "thisYear"              => CallStartsWith(field, now.ToString("yyyy", CultureInfo.InvariantCulture)),
+                "lastYear"              => CallStartsWith(field, now.AddYears(-1).ToString("yyyy", CultureInfo.InvariantCulture)),
+                "nextYear"              => CallStartsWith(field, now.AddYears(1).ToString("yyyy", CultureInfo.InvariantCulture)),
+                "at"                    => Expression.Equal(field, searchValue),
+                "not_at"                => Expression.NotEqual(field, searchValue),
+                "at_before"             => Expression.LessThan(CallCompare(field, searchValue), Expression.Constant(0)),
+                "at_after"              => Expression.GreaterThan(CallCompare(field, searchValue), Expression.Constant(0)),
+                "all"                   => Expression.Constant(true),
+                "true"                  => Expression.IsTrue(field),
```

Comment:
`'true'` and `'false'` are crashing
![image](https://github.com/user-attachments/assets/17ae66a2-b26a-4206-bdda-7202f1b40cc6)

Reply:
It's working now
![image](https://github.com/user-attachments/assets/909ff1aa-dc52-4a2c-a772-217cd99cf93a)